bin/build-rust-docs.sh

```diff
@@ -49,6 +49,9 @@ echo '<meta http-equiv=refresh content=0;url=./adding-glean-to-your-project/inde
 # General API
 echo '<meta http-equiv=refresh content=0;url=../reference/general/index.html>' > build/docs/book/user/general-api.html
 echo '<meta http-equiv=refresh content=0;url=../reference/general/experiments-api.html>' > build/docs/book/user/experiments-api.html
+# General reference
+mkdir -p build/docs/book/reference/general
+echo '<meta http-equiv=refresh content=0;url=toggling-collection-status.html>' > build/docs/book/reference/general/toggling-upload-status.html
```

Comment:
This references `toggling-upload-status.html` but elsewhere you have changed the page title to `toggling-collection-status`

Reply:
That's the redirect. The old `toggling-upload-status.html` will auto-redirect to the new `toggling-collection-status.html>`